src/components/Tooltip/Tooltip.tsx

```diff
@@ -0,0 +1,33 @@
+import clsx from 'clsx'
+import React, { useEffect, useRef, useState } from 'react'
+
+interface Props {
+  text: string
+  className?: string
+  children: React.ReactNode
+}
+
+const Tooltip: React.FC<Props> = ({ text, children, className }) => {
+  const [height, setHeight] = useState(0)
+  const parentWrapperRef = useRef<HTMLDivElement>(null)
+
+  useEffect(() => {
+    if (parentWrapperRef.current) {
+      setHeight(parentWrapperRef.current.clientHeight)
+    }
+  })
```

Comment:
Das fühlt sich falsch an, weil kein dependency array. Das wird also bei jedem Render ausgelöst, richtig? Es müsste doch mit dependency array auch klappen?

```suggestion
  useEffect(() => {
    if (parentWrapperRef.current) {
      setHeight(parentWrapperRef.current.clientHeight)
    }
  }, [parentWrapperRef.current])
```